spring-kafka-docs/src/main/antora/modules/ROOT/pages/retrytopic/how-the-pattern-works.adoc

```diff
@@ -16,8 +16,8 @@ IMPORTANT: You can set the `AckMode` mode you prefer, but `RECORD` is suggested.
 IMPORTANT: At this time this functionality doesn't support class level `@KafkaListener` annotations.
 
 When using a manual `AckMode` with `asyncAcks` set to true, the `DefaultErrorHandler` must be configured with `seekAfterError` set to `false`.
-Starting with versions 2.9.10, 3.0.8, this will be set to true unconditionally for such configurations.
-With earlier versions, it was necessary to override the `RetryConfigurationSupport.configureCustomizers()` method to set the property to `true`.
+Starting with versions 2.9.10, 3.0.8, this will be set to `false` unconditionally for such configurations.
```

Comment:
Was it `false` in those releases?

Reply:
yes, When using a manual `AckMode` with `asyncAcks` set to `true`, will unconditionally set to `false`
see https://github.com/spring-projects/spring-kafka/issues/2707 and https://github.com/spring-projects/spring-kafka/issues/2702.